spec.html

```diff
@@ -32215,26 +32242,88 @@ <h1>
         </emu-note>
       </emu-clause>
 
-      <emu-clause id="sec-defaulttimezone" type="implementation-defined abstract operation">
-        <h1>DefaultTimeZone ( ): a String</h1>
+      <emu-clause id="sec-time-zone-identifier-record">
+        <h1>Time Zone Identifier Record</h1>
+        <p>A <dfn variants="Time Zone Identifier Records">Time Zone Identifier Record</dfn> is a Record used to describe an available named time zone identifier, as well as the primary time zone identifier corresponding to it.</p>
```

Comment:
```suggestion
        <p>A <dfn variants="Time Zone Identifier Records">Time Zone Identifier Record</dfn> is a Record used to describe an available named time zone identifier, as well as the primary time zone identifier of the available named time zone corresponding to it.</p>
```

Reply:
@michaelficarra I think the suggestion below may address the same concern with fewer words. Holler if you want me to integrate your suggestion text instead, otherwise I'll go with the shorter change below.

```suggestion
        <p>A <dfn variants="Time Zone Identifier Records">Time Zone Identifier Record</dfn> is a Record used to describe an available named time zone identifier and its corresponding primary time zone identifier.</p>
```

Reply:
Yes, I think that does it.